loopchain/blockchain/blockchain.py

```diff
@@ -1078,6 +1090,13 @@ def _init_blockchain(self):
             logging.debug("restore from last block hash(" + str(self.__last_block.header.hash.hex()) + ")")
             logging.debug("restore from last block height(" + str(self.__last_block.header.height) + ")")
 
+    def init_crep_reps(self) -> None:
```

Comment:
I think this method aims `init_reps_hash`.

Reply:
Well.... this is only for origin CREPs.